clang/test/Preprocessor/riscv-target-features.c

```diff
@@ -1307,6 +1309,13 @@
 // CHECK-ZVKT-EXT: __riscv_zvkt 1000000{{$}}
 
 // Experimental extensions
+// RUN: %clang --target=riscv32 -menable-experimental-extensions \
+// RUN: -march=rv32i_zaamo0p1 -x c -E -dM %s \
```

Comment:
I'm confused how/if this is actually working. Shouldn't it be zaamo0p2?

Reply:
Sorry for that I forgot to update this part...